### DISCUSSION TITLE
Multiple COUNTRY env variables

### DISCUSSION THREAD

**Discussion by CitizenDev** [jef/streetmerchant#1139] [Help]
Hi.

I set multiple COUNTRY env variables (comma separated) but I'm getting an Unknown country error.
Is only one country supported for nvidia.com at the time?

**Comment by jef** [comment]
Yes, it is. Also if you're in the US, I would not recommend using nvidia as they are not selling cards.

**Comment by CitizenDev** [comment]
I live in European country that does not even have separate Nvidia site, so I need to look various european nvidia sites that may ship to my country. Like germany, france etc
Can I in theory run multiple instances for 3 different countries?

**Comment by jef** [comment]
Nvidia has also changed their API a bit based on this issue: https://github.com/jef/streetmerchant/issues/1135

I would suggest just looking at `nvidia`, rather than `nvidia-api`. `nvidia` doesn't use `COUNTRY`.

But that being said, `nvidia` is only for USA.

**Comment by CitizenDev** [comment]
Thanks for fast replies. Is there a relatively easy way for me to modify current USA nvidia scrape for any other EU one?

**Comment by jef** [comment]
> Thanks for fast replies. Is there a relatively easy way for me to modify current USA nvidia scrape for any other EU one?

There is! Just replace any of the nvidia links that you'd like with your EU ones.

For example:

```
https://www.nvidia.com/en-us/shop/geforce/gpu/?page=1&limit=9&locale=en-us&category=GPU&gpu=RTX%203090
```

would be

```
https://www.nvidia.com/fr-fr/shop/geforce/gpu/?page=1&limit=9&locale=en-us&category=GPU&gpu=RTX%203090
```

**Comment by CitizenDev** [comment]
Great! Which file is it in?

**Comment by JaviAZ** [comment]
src/store/model/nvidia.ts

**Comment by marinovdeyan** [comment]
@jef how can I add links to both fr and gb stores in the `nvidia.ts` file?

also, since the FR and the GB website are in different languages, how can I add both languages for the `inStock` bit?
